<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferDialog</class>
 <widget class="QDialog" name="TransferDialog">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>400</width><height>280</height></rect>
  </property>
  <property name="windowTitle"><string>Transfer Funds</string></property>
  <widget class="QWidget" name="widget">
   <widget class="QSpinBox" name="fromIdSpinBox"><property name="geometry"><rect><x>150</x><y>20</y><width>200</width><height>30</height></rect></property></widget>
   <widget class="QSpinBox" name="toIdSpinBox"><property name="geometry"><rect><x>150</x><y>60</y><width>200</width><height>30</height></rect></property></widget>
   <widget class="QDoubleSpinBox" name="amountDoubleSpinBox"><property name="geometry"><rect><x>150</x><y>100</y><width>200</width><height>30</height></rect></property><property name="maximum"><double>1000000</double></property></widget>
   <widget class="QPushButton" name="transferButton"><property name="geometry"><rect><x>150</x><y>150</y><width>100</width><height>40</height></rect></property><property name="text"><string>Transfer</string></property></widget>
   <widget class="QPushButton" name="backButton"><property name="geometry"><rect><x>20</x><y>200</y><width>100</width><height>30</height></rect></property><property name="text"><string>Back</string></property></widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
